### DISCUSSION TITLE
Give simple access to toast notification

### DISCUSSION THREAD

**Discussion by SenorRagequit** [exorstudios/riftbreaker-tools#154] [Wishlist]
**Wish**
---
A way to access the toast notification for modding:
![grafik](https://user-images.githubusercontent.com/50502395/179426118-098b9827-6b45-4f80-90a3-f00ab2eb4bc6.png)

**Why**
---
It is a simple, not very distracting way of sending information.

**Example**
---
I thought about:
`GuiService:SendToastNotification( Text, Color [red/green/blue], duration [seconds])`
Like that, we could add information if f.e a mod happens to trigger "INVALID_ID" we might add some information in here, like
`GuiService:SendToastNotification( "Int should be bigger than 0", red, 5)`

**Comment by lukaasm** [comment]
![image](https://user-images.githubusercontent.com/48881/180780842-43aec2cd-7a2a-43c4-825f-54944c96a5eb.png)
for now generic one without duration/style
`
GuiService:ShowHudPickupText("text")
`
